nbs/core.ipynb

```diff
@@ -57,7 +57,7 @@
     "import os\n",
```

Comment:
<p>I think the comparison between <code>initial_weights</code> and <code>after_weights</code> might be missing.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>